src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private final FruitDao fruitDao;
+
+    public ReportServiceImpl(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder("fruit,quantity");
```

Comment:
It is a special service for reports, it would be good to move the header to constants